content/docs/0.18.x/define/quality-gates/index.md

```diff
@@ -1,24 +1,131 @@
 ---
-title: Working and Integrating with Quality Gates
-description: Integrate a Quality Gate into an existing pipeline.
-weight: 7
+title: Quality Gates
+description: Implement Quality Gates evaluations in your project
+weight: 70
 keywords: [0.18.x-quality_gates]
 ---
 
+A quality gate allows you to validate a deployment or release
+using data delivered by your observability platform
+and using quality criteria that you define as Service Level Objectives (SLOs).
```

Comment:
Not sure, but are SLOs defined somewhere?

Reply:
Yup, we have a (nascent) reference page for SLO's.  Good catch -- I added a link to that page.